ui/src/main/java/edu/wpi/grip/ui/pipeline/StepController.java

```diff
@@ -192,6 +206,25 @@ private void moveStepRight() {
     pipeline.moveStep(step, +1);
   }
 
+  @Subscribe
+  @SuppressWarnings("PMD.UnusedPrivateMethod")
```

Comment:
There must be a way to disable this error when there is this annotation on the method.
Not sure how to do it.